chapters/en/chapter2/tts_pipeline.mdx

```diff
@@ -0,0 +1,98 @@
+# Audio generation with a pipeline
+
+Audio generation encompasses a versatile set of tasks that involve producing an audio output. The tasks 
+that we will look into here are speech generation (aka Text-to-speech task) and music generation. In text-to-speech, a 
+model transforms a piece of text into lifelike spoken language sound, opening the door to applications such as virtual assistants, 
+accessibility tools for the visually impaired, and personalized audiobooks. 
+On the other hand, music generation can enable creative expression, and finds its use mostly in entertainment and game 
+development industries. 
+
+In 🤗 Transformers, you'll find a pipeline that covers both of these tasks. This pipeline is called `"text-to-audio"`, 
+but for convenience, it also has a `"text-to-speech"` alias. Here we'll use both, and you are free to pick whichever 
+seems more descriptive for your task. 
+
+Let's explore how you can use this pipeline to start generating audio narration for texts, and music with just a few lines of code.
+
+This pipeline is new to 🤗 Transformers, thus you'll need to install the library from the source: 
+
+```bash
+pip install git+https://github.com/huggingface/transformers.git
+```
+
+## Generating speech
+
+Let's begin by exploring text-to-speech generation. First, just as it was the case with audio classification and automatic 
+speech recognition, we'll need to define the pipeline. We'll use the `suno/bark-small` model with this pipeline:
```

Comment:
Always nice linking the model cards IMO so the user can read more about the checkpoint!
```suggestion
speech recognition, we'll need to define the pipeline. We'll define a text-to-speech pipeline since it best describes our task, and use the [`suno/bark-small`](https://huggingface.co/suno/bark-small) checkpoint:
```